docs/cuda-miner.md

```diff
@@ -213,6 +213,29 @@ by the node.
 *Note:* If you are combining `repeats` and `instances`, it is the *number of repeats per instance* that is configured! I.e. with 2
 instances and repeats 5 each GPU will run 5 attempts per run.
 
+### Multiple GPU miners
+
+If you want to address different GPU-cards with different miners' configurations, you can set up multiple miners in the config.
+For example, if you have 4 GPU-cards, and you want to address 0 and 1 with different config than 2 and 3, you can use the following config:
+```yaml
+mining:
+    cuckoo:
+        edge_bits: 29
```

Comment:
this line is breaking check_config

Reply:
I don't know how this would be possible, let's chat offline..

Reply:
it's OK now